README.md

```diff
@@ -406,23 +406,26 @@ More information here: [Environment variables](https://docs.appdynamics.com/22.2
 
 Please note that AppDynamics requires Mendix 6.2 or higher.
```

Comment:
The buildpack only supports Mendix >= 7.

Reply:
Changed to 7.15. As in main documentation